src/CFamily.UnitTests/packages.lock.json

```diff
@@ -1556,7 +1556,8 @@
         "dependencies": {
           "Microsoft.VisualStudio.Sdk": "[17.0.31902.203, )",
```

Comment:
Most files are package updates so you can ignore those, needed to add a reference to IssueVisualization.

Reply:
I don't think `CloudSecrets` should depend on `IssueVisualization` - they are separate feature areas.
It would be better to move `ILineHashCalculator` to `Infrastructure.VS` since it is a fairly low-level helper class (it can't go in `Core` because is uses depends on VS types).

Reply:
Could you move the interface and implementation to `Infrastructure.VS`, and put them in a new folder called `Editor` please? (there are a couple of editor-related helpers in `Infrastructure.VS` so it would be good to have a folder we can use to group them at some point).